### PR TITLE
feature/104: Added `toBoolMapping` mapping type

### LINKED ISSUE #104: Add toBoolMapping
Add toBoolMapping

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #104

### CODE REVIEW

File: src/main/java/com/exasol/adapter/document/edml/ToDoubleMapping.java [hunk 1]
```diff
@@ -5,14 +5,13 @@
...
- * Java representation of the EDML {@code toDecimalMapping}.
+ * Java representation of the EDML {@code toDoubleMapping}.
  */
 @EqualsAndHashCode(callSuper = true)
 @ToString(callSuper = true)
 @Jacksonized
 @SuperBuilder
 @Data
-@SuppressWarnings("java:S1170") // sonar can't deal with Lombok
 public final class ToDoubleMapping extends AbstractToNumberMapping {
```
Comment (morazow) [inline src/main/java/com/exasol/adapter/document/edml/ToDoubleMapping.java:15]: It is not a problem here for sonar?

File: src/main/java/com/exasol/adapter/document/mapping/PropertyToBoolColumnMapping.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package com.exasol.adapter.document.mapping;
+
+import com.exasol.adapter.metadata.DataType;
+
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * This class defines a mapping that extracts a decimal number from the remote document and maps it to an Exasol
+ * {@code DECIMAL} column.
```
Comment (morazow) [inline src/main/java/com/exasol/adapter/document/mapping/PropertyToBoolColumnMapping.java:10]:
```suggestion
 * This class defines a mapping that extracts a boolean value from the remote document and maps it to an Exasol
 * {@code BOOLEAN} column.
```

**Review by morazow — CHANGES_REQUESTED** [review]
Added some minor suggestions

File: src/main/java/com/exasol/adapter/document/edml/ToDoubleMapping.java (hunk 1 above)
Comment (jakobbraun) [reply src/main/java/com/exasol/adapter/document/edml/ToDoubleMapping.java:15]: no. It only can't deal with the dafault values.

**Review by morazow — APPROVED** [review]

**Pull request merged**